ts/api/pagopa.ts

```diff
@@ -134,12 +134,80 @@ type GetWalletsUsingGETExtraT = MapResponseType<
   WalletListResponse
 >;
 
+/**
+ *
+ * sanitaze psp tags if it is an array with values (avoid no string value and duplicates)
+ * otheriwise set tags as an empty array
+ * @param w the wallet object
+ */
+const fixWalletPspTagsValues = (w: any) => {
+  if (Array.isArray(w.psp.tags)) {
+    // tslint:disable-next-line: readonly-array
+    const duplicates: string[] = [];
+    // tslint:disable-next-line: no-object-mutation
+    w.psp = {
+      ...w.psp,
+      tags: w.psp.tags.filter((item: any) => {
```

Comment:
you don't need the `duplicates` array to deduplicate

```suggestion
      tags: w.psp.tags
        .filter((item: any, idx) => typeof item === "string" && w.psp.tags.indexOf(item) === idx)
```